### ISSUE TITLE
Hacknight Info

### ISSUE THREAD

**Issue by benjaminreid** [nouveller/twilio-hack#1]
## Info
- Name: Slack SMS
- Description: Connect with your Slack team over SMS
- Rails app: https://github.com/nouveller/twilio-hack, Node app: https://github.com/paulspringett/slacksms-node
- **no live version**
## Tech

We used a Rails app to authorise with Slack and get the users phone number, this then talked to a Node app which used Twilio to send you a text message to kick things off. From there you text the the number back which Twilio tells the Node app about, which then updates Slack via their API. Thus, allowing you to post messages to Slack over SMS.
## Screenshots

![1](https://cloud.githubusercontent.com/assets/713128/7200832/acfa2326-e4f8-11e4-8e8e-ef34a47b81ed.jpg)
![2](https://cloud.githubusercontent.com/assets/713128/7200827/acd3e350-e4f8-11e4-9422-f8ede928961b.jpg)
![3](https://cloud.githubusercontent.com/assets/713128/7200831/acf928ae-e4f8-11e4-8dc1-8fa11f002439.jpg)
![4](https://cloud.githubusercontent.com/assets/713128/7200828/ace80d08-e4f8-11e4-8cef-b085e1dee358.jpg)
![5](https://cloud.githubusercontent.com/assets/713128/7200829/ace9bf4a-e4f8-11e4-9d2f-023a522f4a1e.jpg)
![6](https://cloud.githubusercontent.com/assets/713128/7200830/acf786a2-e4f8-11e4-9607-5029acacd1df.jpg)

**Comment by benjaminreid** [comment]
@paulspringett, @daviddarnes, @studioromeo have anything to add/change?

It's for a blogpost on the Simpleweb blog.

**Comment by daviddarnes** [comment]
Nope! Minor tweaks but I like that this was all achieved in the time allocated.
